tests/integration/specs/common/autostore.spec.js

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+import nock from 'nock';
+import { init, DataStore, DataStoreType, User, Query } from '__SDK__';
+import { collectionName } from '../config';
+import { cleanUpAppData } from '../utils';
+
+describe('AutoStore', function() {
+  const cachedDocs = [];
+
+  before(function () {
+    // Turn off nock
+    return nock.restore();
+  });
+
+  before(function() {
+    // Initialize the SDK
+    return init({
+      appKey: process.env.APP_KEY,
+      appSecret: process.env.APP_SECRET
+    });
+  });
+
+  before(function () {
+    // Remove any existing data in the collection
+    return cleanUpAppData(collectionName);
+  });
+
+  before(function() {
+    // Signup an anonymous user
+    return User.signup();
```

Comment:
If we record the userId we can pass it to cleanUpAppData where it would be deleted and we would avoid the explicit deletion of the user in the '**after**' hook below.

Reply:
This has been changed to the recommendation above.